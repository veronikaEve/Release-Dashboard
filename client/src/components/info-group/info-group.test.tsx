import { render } from "@testing-library/react";

import InfoGroup from "./info-group";

describe("InfoGroup", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<InfoGroup />);
    expect(asFragment()).toMatchSnapshot();
  });
});
