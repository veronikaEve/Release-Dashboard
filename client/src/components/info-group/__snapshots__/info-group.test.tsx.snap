// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoGroup should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="info-group row"
  >
    <div
      class="info-group__label col-md-4"
    >
      Name:
    </div>
    <div
      class="info-group__info col-md-8"
    >
      Wazzaaaap
    </div>
  </div>
</DocumentFragment>
`;
